### ISSUE TITLE
Shadows not Appearing during Export as PDF

### ISSUE THREAD

**Issue by kylekozielski** [jgraph/drawio#4613]
Exactly what the title states. I am using this on google chrome and other colleagues of mine also have the same issue. 

The shadows appear in the diagram, but the moment I export as a PDF, they disappear. They do not disappear when exporting as other file types.

**Comment by davidjgraph** [comment]
This is expected. We cannot convert the shadow to vector so exclude it.

**Comment by kylekozielski** [comment]
> This is expected. We cannot convert the shadow to vector so exclude it.

But, this exact function used to work about a month ago?

**Comment by davidjgraph** [comment]
Yes. we switched it off for performance problems.

**Comment by jesseishi** [comment]
Are there plans to be able to export shadows to vector for a future release?

**Comment by jesseishi** [comment]
Thank you for your answer. Could it maybe be included in the desktop version?

**Comment by davidjgraph** [comment]
it can't because that's chrome.

**Comment by jesseishi** [comment]
Clear. Thank you!

**Comment by jesseishi** [comment]
I've contacted the Chrome development team through their issue tracker. They would need more information from someone from the draw.io development team to understand what they would need to change to support this. The issue that I reported can be found here ([https://issues.chromium.org/issues/365033303](https://issues.chromium.org/issues/365033303)) and it would be super nice if someone from the draw.io team could maybe comment there or make a new issue so the Chrome team can work to resolve this. Thanks!
